lib/scrabble.rb

```diff
@@ -1,9 +1,24 @@
 class Scrabble
 
-  def score(word)
-    1
+  def score(word = nil)
+    if word == nil || word == ""
+      return 0
+    else
+      letters = word.upcase.split('')
+      value = letters.map { |letter| point_values.values_at(letter)[0] }
+      tally = value.inject(:+)
```

Comment:
Take a look at the expanded syntax for inject. You can combine the map above with the inject you've used to make one enumerable.